backend/cfg/cfg_irc.ml

```diff
@@ -330,10 +330,10 @@ let assign_colors : State.t -> Cfg_with_layout.t -> unit =
       n.Reg.irc_color <- alias.Reg.irc_color)
 
 type direction =
-  | Load_before_cell of Cfg.BasicInstructionList.cell
-  | Store_after_cell of Cfg.BasicInstructionList.cell
-  | Load_after_list of Cfg.BasicInstructionList.t
-  | Store_before_list of Cfg.BasicInstructionList.t
+  | Load_before_cell of Cfg.basic Cfg.instruction Cfg.DoublyLinkedList.cell
```

Comment:
`Cfg.basic Cfg.instruction Cfg.DoublyLinkedList.cell` is long, what do you think about adding a name/alias for it, similar to `Cfg.basic_instruction_list`?